e2e/specs/planner.e2e-spec.ts

```diff
@@ -0,0 +1,194 @@
+        sessionsToCreate = [{
+            id: uuid(),
+            userTransactionId: uuid(),
+            personId: null,
+            roomId: null,
+            duration: 3600,
+            start: today,
+            sessionTypeCode: SessionTypesCodes[SessionTypes.FTRACK_TRIAL_ONLY]
+        },
+            {
+                id: uuid(),
+                userTransactionId: uuid(),
+                personId: JudgesCodes[Judges.AMY_WESSOME],
+                roomId: null,
+                duration: 3600,
+                start: today,
+                sessionTypeCode: SessionTypesCodes[SessionTypes.FTRACK_TRIAL_ONLY]
+            },
+            {
+                id: uuid(),
+                userTransactionId: uuid(),
+                personId: JudgesCodes[Judges.AMY_WESSOME],
+                roomId: RoomCodes[Rooms.COURT_4],
+                duration: 3600,
+                start: today.clone().minute(15),
+                sessionTypeCode: SessionTypesCodes[SessionTypes.FTRACK_TRIAL_ONLY]
+            }, {
+                id: uuid(),
+                userTransactionId: uuid(),
+                personId: JudgesCodes[Judges.JUDGE_LINDA],
+                roomId: null,
+                duration: 3600,
+                start: today.clone().minute(15),
+                sessionTypeCode: SessionTypesCodes[SessionTypes.FTRACK_TRIAL_ONLY]
+            }
+        ];
+    });
+
+    describe('Check if newly created sessions are on planner ', () => {
+        it('Create sessions ', async () => {
+            numberOfVisibleEvents = await plannerPage.getNumberOfVisibleEvents();
+            expect(numberOfVisibleEvents).not.toEqual(0);
+            for (let session of sessionsToCreate) {
+                expect(await API.createSession(session)).toEqual(200);
+                Logger.log('Created session: id: ' + session.id + ' start: ' + session.start.toISOString());
+            }
+        });
+
+        it('Refresh the view ', async () => {
+            await plannerPage.clickNextButton();
+            await plannerPage.clickPrevButton();
+        });
+
+        it('Refreshed view should show new sessions ', async () => {
+            let newNumberOfVisibleEvents = await plannerPage.getNumberOfVisibleEvents();
+            visibleElementsDiff = Math.abs(newNumberOfVisibleEvents - numberOfVisibleEvents);
+            Logger.log('Sessions number, before call: ' +
+                numberOfVisibleEvents + ' after: ' + newNumberOfVisibleEvents +
+                ' diff: ' + visibleElementsDiff);
+
+            expect(visibleElementsDiff).toEqual(sessionsToCreate.length);
+        });
+
+        it('There should be at least one not allocated Session', async () => {
+            plannerPage.waitUntilVisible();
+            resourceId = await plannerPage.getResourceIdByName('Not allocated');
+            Logger.log('"Not allocated" resource id: ' + resourceId);
+
+            textToCheck = 'No Room - No Judge - ' + SessionTypes.FTRACK_TRIAL_ONLY;
```

Comment:
Isn't that check too strict? I think the format of the text should be unit tested and here we should verify if the components of text (ie: 'No Room'; 'No Judge'') exists within the event. With your approach modifying this message's format would brake the tests :(